torchgeo/losses/__init__.py

```diff
@@ -3,6 +3,8 @@
 
 """TorchGeo losses."""
 
+from .focaljaccard import BinaryFocalJaccardLoss
 from .qr import QRLoss, RQLoss
+from .xentjaccard import BinaryXEntJaccardLoss
 
-__all__ = ('QRLoss', 'RQLoss')
+__all__ = ('QRLoss', 'RQLoss', 'BinaryFocalJaccardLoss', 'BinaryXEntJaccardLoss')
```

Comment:
Never seen these losses before, are they standard in the literature? If not, I don't think we should add them.